docs/library/forms/textfield.md

```diff
@@ -57,22 +73,61 @@ class TextfieldControlled(rx.State):
 def controlled_example():
     return rx.vstack(
         rdxt.heading(TextfieldControlled.text),
-        rdxt.textfield_root(
-    rdxt.textfield_slot(
-        rdxt.icon(tag="magnifying_glass"),
-    ),
-    rdxt.textfield_input(
-        placeholder="Search here...",
-        value=TextfieldControlled.text,
-        on_change=TextfieldControlled.set_text,
-    ),
-    
-)
+        rdxt.input(
+            icon="magnifying_glass", 
+            placeholder="Search here...", 
+            value=TextfieldControlled.text,
+            on_change=TextfieldControlled.set_text,
+        ),
     )
 ```
 
 
-# Real World Example
+
+Behind the scene, the input component is implemented using debounced input to avoid sending individual state updates per character to the backend while the user is still typing. This allows a state var to directly control the `value` prop from the backend without the user experiencing input lag. For advanced use cases, you can tune the debounce delay by setting the `debounce_timeout` when creating the Input component. You can find examples of how it is used in the [DebouncedInput]({library.forms.debounce.path}) component.
+
+
+
+### Submitting a form using input
+
+The `name` prop is needed to submit with its owning form as part of a name/value pair.
+
+When the `required` prop is `True`, it indicates that the user must input text before the owning form can be submitted.
+
+The `type` is set here to `password`. The element is presented as a one-line plain text editor control in which the text is obscured so that it cannot be read. The `type` prop can take any value of [`button`, `checkbox`, `color`, `date`, `datetime-local`, `email`, `file`, `hidden`, `image`, `month`, `number`, `password`, `radio`, `range`, `reset`, `search`, `submit`, `tel`, `text`, `time`, `url`, `week`]. Learn more [here](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/input/password).
```

Comment:
I don't think we need to list them all like this, a link should be enough. Also, the link should just go to /input rather than /input/password